vision-zero

```diff
@@ -0,0 +1,339 @@
+#!env python
```

Comment:
oo another thing to note is i had to change this shebang to `#!/usr/bin/env python` for it to work on my machine. im not really sure why i need to use an absolute path for it to work and yall dont tho :0

Reply:
that is a good question! i would have expected `source`ing the `activate` script out of the venv would have set those paths. Thanks for this extra piece of info though - I can't wait to iron out these wrinkles tomorrow.

Reply:
I addressed this issue through testing and experimentation on an x86 machine. Thank you Rose!